regolith/helpers/a_presentationhelper.py

```diff
@@ -145,7 +151,35 @@ def db_updater(self):
                     wait_bool = add_to_google_calendar(event)
                     jump += 1
 
+        # if gitlab box is checked, creates gitlab repo in talks group
+
+        if rc.gitlab:
+            headers = {
+                # TODO : store this in environment variable somehow
+                'PRIVATE-TOKEN': '<api_authentication_key>'
+            }
+
+            # namespace_id = 35 always because talks group id is 35
+            # name derived by combining last 2 digits of the begin_date year, the begin_date month, first 2
+            # letters of person, underscore, place
+
+            # TODO : assuming begin_date is in correct format and follows yyyy-mm-dd
+            params = {
+                'name': rc.begin_date.strip()[2:4] + rc.begin_date.strip()[5:7] + rc.person.strip()[0:2]
+                        + '_' + rc.place.strip(),
+                'namespace_id': '35',
```

Comment:
this shouldn't be hard-coded.  Somehow all the information about the URL of the target repo and the `namespace_id` should be stored in the run-control json file, like the database addresses themselves.

Reply:
I am having some trouble figuring out how to do this! I haven't worked with json files and API requests.